sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1376,6 +1381,11 @@ def task_worker(self):
                     # Skip if these essential routines are not available
                     self.update_port_transceiver_status_table_sw_cmis_state(lport, CMIS_STATE_READY)
                     continue
+                except Exception as e:
```

Comment:
@mihirpat1 why we made distinction between AttributeError vs others?

Reply:
@prgeor The AttributeError relevant handling here seems to be a day 1 code. However, I am not sure on the original reason for this so decided to leave the existing behavior as is since with AttributeError, we are moving CMIS SM to `CMIS_STATE_READY`.
Hence, I am now handling all other exceptions in a different block and moving the CMIS SM to `CMIS_STATE_FAILED` state.